<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
     <item>
       <widget class="QPushButton" name="_btn_state_playback">
        <property name="text">
         <string>Play/Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_reset_playback">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_integrated">
        <property name="text">
         <string>Detect and Move dog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_detect">
        <property name="text">
         <string>Detect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_spinLabel">
        <property name="text">
         <string>Pos num to try</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_spinBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_reach">
        <property name="text">
         <string>Reachability</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_calcpath">
        <property name="text">
         <string>Calculate Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_scan">
        <property name="text">
         <string>Get Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_im">
        <property name="text">
         <string>Get Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_im_label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
